homeassistant/components/media_player/__init__.py

```diff
@@ -616,6 +618,11 @@ def source_list(self):
         """List of available input sources."""
         return None
 
+    @property
+    def channel_list(self):
```

Comment:
Has this been discussed in an issue in our architecture repo?

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model

Reply:
@MartinHjelmare 
No, I should have, but didn't understand it when I read that paragraph...
I will

Reply:
Here it is : https://github.com/home-assistant/architecture/issues/62